kolibri/plugins/coach/assets/src/vue/coach-exercise-render-page/answer-history/index.vue

```diff
@@ -0,0 +1,156 @@
+<template>
```

Comment:
Minor note - typically for subcomponents we don't make a subdir, and this would just be called

    coach-exercise-render-page/answer-history.vue